### PR TITLE
Assign properties from Tags automatically

### PR DESCRIPTION
**Pull request by a1exsh**
Retrying from scratch after rebase on newer master.

### CODE REVIEW

File: zmon-agent.py [hunk 1]
```diff
@@ -60,11 +59,12 @@ def get_hash(ip):
 def get_tags_dict(tags):
     return { t['Key']: t['Value'] for t in tags }
 
-def assign_stack_name_and_version_from_tags(obj, tags):
-    if 'StackName' in tags:
-        obj['stack_name'] = tags['StackName']
-    if 'StackVersion' in tags:
-        obj['stack_version'] = tags['StackVersion']
+def assign_properties_from_tags(obj, tags):
+    import inflection
```
Comment (hjacobs) [inline zmon-agent.py]: I like your inflection approach :smile:

Comment (hjacobs) [inline zmon-agent.py]: Maybe we need to check that there are no collisions? What about tags like "id" etc possibly overwriting other fields.

Comment (a1exsh) [inline zmon-agent.py]: It is checked on the next line: `if key not in obj:`

**Comment by feikesteenbergen** [comment]
+1, especially great for Spilo, as all it's resources are tagged with the tag SpiloVersion.

**Pull request merged**